src/isIsogram.test.js

```diff
@@ -6,4 +6,24 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`enters izogram`, () => {
+    expect(isIsogram('horse')).toBe(true);
+  });
+
+  it(`enters not izogram`, () => {
+    expect(isIsogram('mathematic')).toBe(false);
+  });
+
+  it(`enters with Uppercase not izogram`, () => {
+    expect(isIsogram('Clock')).toBe(false);
+  });
+
+  it(`enters with Uppercase izogram`, () => {
+    expect(isIsogram('Term')).toBe(true);
+  });
+
+  it(`enters with Uppercase izogram`, () => {
```

Comment:
The description `enters with Uppercase izogram` is duplicated from the previous test case. Consider renaming it to something more descriptive, like `returns true for empty string`.